### PR TITLE
Texts in configuration

### PR DESCRIPTION
**Pull request by stephanvierkant**
Disclaimer: my first pull request ever that contains TypeScript, so I hope I haven't done something completely wrong.

### CODE REVIEW

**Comment by ivoviz** [comment]
@sahlhoff @stephanvierkant  Sorry for the delay, I'm really busy lately.

@stephanvierkant  Would it be possible to move `footnote` into `lang` as well, and rename `lang` to something like `texts`?

**Comment by sahlhoff** [comment]
@stephanvierkant @ivoviz sorry to piggyback off of this PR but would it be possible to make the headers also configurable? Would be useful for security reasons. I could make this in a separate PR too.

**Comment by ivoviz** [comment]
@sahlhoff Sure! I'm not sure what options people need, so at the moment it's really just the basic reimplementation of the old version.

**Comment by stephanvierkant** [comment]
@ivoviz @sahlhoff I totally forgot about this PR, but I finally updated it. WDYT?

File: lib/feedback.ts [hunk 1]
```diff
@@ -87,19 +87,42 @@ interface Helper extends Area {
...
     allowedTags: [
       'button', 'a', 'span', 'h1', 'h2', 'h3', 'h4', 'h5', 'p',
       'i', 'strong', 'small', 'sub', 'sup', 'b', 'time', 'img',
       'caption', 'input', 'label', 'legend', 'select', 'textarea',
       'details', 'summary'
     ],
-    footnote:
+    endpoint: 'https://very-api-so-cool.url/',
+    classes: {
+      'prefix': 'fb-',
```
Comment (ivoviz) [inline lib/feedback.ts]: Nit: Could you remove the `'`s from the property names in `classes` and `texts`?

Comment (stephanvierkant) [reply lib/feedback.ts]: I'll do!

Comment (stephanvierkant) [reply lib/feedback.ts]: Done!

**Comment by ivoviz** [comment]
Thanks!

**Comment by stephanvierkant** [comment]
You're welcome, sorry for the delay. Can you tag/publish a new version?

**Pull request merged**